docusaurus/docs/create-a-plugin/extend-a-plugin/fetch-data-from-frontend.md

```diff
@@ -0,0 +1,287 @@
+---
+id: fetch-data-from-frontend
+title: Fetching data from frontend code
```

Comment:
Think this should include proxy. Otherwise would expect this page to discuss fetching data from a backend plugin as well.